<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>750</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>590</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
    </font>
   </property>
   <property name="text">
    <string>摄像头识别</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/Administrator/yolov5-7.0/UI设计/images/摄像头_camera-one.png</normaloff>C:/Users/Administrator/yolov5-7.0/UI设计/images/摄像头_camera-one.png</iconset>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>540</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>图片识别</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/Administrator/yolov5-7.0/UI设计/images/图片收集_collect-picture.png</normaloff>C:/Users/Administrator/yolov5-7.0/UI设计/images/图片收集_collect-picture.png</iconset>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>540</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: qlineargradient(spread:pad, x1:0.0227727, y1:0.472, x2:0.977273, y2:0.476955, stop:0 rgba(23, 98, 255, 255), stop:1 rgba(255, 150, 207, 255));
	font: 75 16pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);
border-radius: 20
}
QPushButton:hover{
background-color:qlineargradient(spread:pad, x1:0.198909, y1:0.302, x2:0.892046, y2:0.687, stop:0 rgba(23, 98, 255, 255), stop:1 rgba(255, 150, 207, 255));
padding-bottom: 7;

}
QPushButton:pressed{
padding-top: 7;
padding-left: 7;
}
QPushButton:disabled{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(119, 119, 119, 255), stop:1 rgba(221, 221, 221, 255));
	color: rgba(241, 241, 241, 150);
}</string>
   </property>
   <property name="text">
    <string>调用detect</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>380</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 11pt &quot;微软雅黑&quot;;
background-color: qlineargradient(spread:pad, x1:0.0227727, y1:0.472, x2:0.977273, y2:0.476955, stop:0 rgba(23, 98, 255, 255), stop:1 rgba(255, 150, 207, 255));
border-radius:20</string>
   </property>
   <item>
    <property name="text">
     <string>best_gray</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>C:/Users/Administrator/.designer/backup/images/logo.png</normaloff>C:/Users/Administrator/.designer/backup/images/logo.png</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>face</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>C:/Users/Administrator/.designer/backup/images/logo.png</normaloff>C:/Users/Administrator/.designer/backup/images/logo.png</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>yolov5l</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>C:/Users/Administrator/.designer/backup/images/logo.png</normaloff>C:/Users/Administrator/.designer/backup/images/logo.png</iconset>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>540</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: qlineargradient(spread:pad, x1:0.0227727, y1:0.472, x2:0.977273, y2:0.476955, stop:0 rgba(23, 98, 255, 255), stop:1 rgba(255, 150, 207, 255));
	font: 75 16pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);
border-radius: 20
}
QPushButton:hover{
background-color:qlineargradient(spread:pad, x1:0.198909, y1:0.302, x2:0.892046, y2:0.687, stop:0 rgba(23, 98, 255, 255), stop:1 rgba(255, 150, 207, 255));
padding-bottom: 7;

}
QPushButton:pressed{
padding-top: 7;
padding-left: 7;
}
QPushButton:disabled{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(119, 119, 119, 255), stop:1 rgba(221, 221, 221, 255));
	color: rgba(241, 241, 241, 150);
}</string>
   </property>
   <property name="text">
    <string>开始识别</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>650</width>
     <height>700</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:30;
background-color: rgba(255, 255, 255, 150);
border-image: url(:/images/images/bg.png);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover{
padding-bottom: 7;
background-color: rgba(255, 255, 255, 0);
}
QPushButton{
background-color: rgba(255, 255, 255, 0);
	color: rgb(0, 170, 255);
	font: 12pt &quot;微软雅黑&quot;;
}
</string>
   </property>
   <property name="text">
    <string>帮助</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>images/帮助_help.png</normaloff>images/帮助_help.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>10</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover{
padding-bottom: 7;
background-color: rgba(255, 255, 255, 0);
}
QPushButton{
background-color: rgba(255, 255, 255, 0);
	color: rgb(0, 170, 255);
	font: 12pt &quot;微软雅黑&quot;;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>images/logo.png</normaloff>images/logo.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>60</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: qlineargradient(spread:pad, x1:0.0227727, y1:0.472, x2:0.977273, y2:0.476955, stop:0 rgba(23, 98, 255, 255), stop:1 rgba(255, 150, 207, 255));
	font: 40 12pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);
border-radius: 20
}
QPushButton:hover{
background-color:qlineargradient(spread:pad, x1:0.198909, y1:0.302, x2:0.892046, y2:0.687, stop:0 rgba(23, 98, 255, 255), stop:1 rgba(255, 150, 207, 255));
padding-bottom: 7;

}
QPushButton:pressed{
padding-top: 7;
padding-left: 7;
}
QPushButton:disabled{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(119, 119, 119, 255), stop:1 rgba(221, 221, 221, 255));
	color: rgba(241, 241, 241, 150);
}</string>
   </property>
   <property name="text">
    <string>返回选择界面</string>
   </property>
  </widget>
  <zorder>label_3</zorder>
  <zorder>radioButton</zorder>
  <zorder>radioButton_2</zorder>
  <zorder>pushButton</zorder>
  <zorder>comboBox</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>pushButton_4</zorder>
  <zorder>pushButton_5</zorder>
  <zorder>pushButton_3</zorder>
 </widget>
 <resources>
  <include location="resouce.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
